src/main/java/UseCase/CreatePlaylistInteractor.java

```diff
@@ -0,0 +1,49 @@
+package UseCase;
+import Database.playlistDsData;
+import Entities.Playlist;
+import InputBoundary.newPlaylistInputBoundary;
+import InputData.newPlaylistInputData;
+import Database.playlistAccessInterface;
+import Database.playlistLibrary;
+import OutputBoundary.newPlaylistOutputBoundary;
+import OutputData.newPlaylistOutputData;
+
+public class CreatePlaylistInteractor implements newPlaylistInputBoundary {
+        private newPlaylistOutputBoundary outputBoundary;
+//        private String message;
+    public CreatePlaylistInteractor(newPlaylistOutputBoundary outputBoundary){
+        this.outputBoundary = outputBoundary;
```

Comment:
Not wrong at all but if you wanted to make this clearer to yourself you could rename outputBoundary to Presenter (since thats what would be passed).